docs/results.rst

```diff
@@ -19,16 +19,30 @@ configurations for this result are:
 
 -  filename: path to the file. Depending on the file extension this file
    will be saved as excel (xls or xlsx), tsv or csv.
+-  encoding: Defaults to 'utf-8'.
+-  index: Write index. True by default.
+-  header: Write column names. True by default.
+-  float_format: example_.
```

Comment:
Not sure if this is a good example, it doesn't use to_csv/to_excel, and it doesn't mention old-style formatting (ex: `%d`)